dwst/scripts/particles.js

```diff
@@ -159,12 +183,10 @@ function readInstructionName(remainder1) {
 function readInstructionArg(remainder1) {
   const nextBreakIndex = indexOfAny(remainder1, [' ', ',', ')']);
   if (nextBreakIndex === 0) {
-    const msg = `broken particle argument: missing argument, remainder = ${remainder1}`;
-    throw new InvalidParticles(msg);
+    throw new InvalidParticles(remainder1, 'argument', 1);
   }
   if (nextBreakIndex === -1) {
-    const msg = `Expected ' or ), remainder = ${remainder1}`;
-    throw new InvalidParticles(msg);
+    throw new InvalidParticles(remainder1, '\' or )', 1);
```

Comment:
Use double quotes here so you don't need to escape single quote

Reply:
check